server/lib/util.py

```diff
@@ -37,6 +37,7 @@
 TOPIC_PAGE_CONFIGS = {
     'equity': ['USA', 'CA'],
     'poverty': ['USA', 'India'],
+    'disasters': ['AK']
```

Comment:
Is this just for testing purposes? adding this creates a topic page for the topic disasters. If you're adding a new config for disaster dashboards, just need to add the file under server/config/disaster_dashboard like: https://github.com/datacommonsorg/website/blob/master/server/config/disaster_dashboard/Earth.textproto

If this config is just for testing purposes, maybe don't include it in this PR?

Reply:
Removed!